docs/build/dapp/smart-contracts/remix-deploy.md

```diff
@@ -49,13 +49,16 @@ To switch to the **Fuji test network**, go to Settings, select Advanced, and the
 
 ## Step 2: Funding Your C-Chain Address
 
-### **Using Avalanche Wallet**
+### **Using Core web**
```

Comment:
```suggestion
### **Using Core Web**
```

Reply:
@ashucoder9 That's a valid suggestion, I was conflicted on this one too. We use 'Core web' in our support docs https://support.avax.network/en/?q=core+web . I thought I should keep the same pattern here. What do you think I should do?